WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsJetpackSocialContainerComposeView.kt

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.ui.posts
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.AbstractComposeView
+import org.wordpress.android.ui.compose.components.TrainOfIconsModel
+import org.wordpress.android.ui.compose.theme.AppThemeEditor
+import org.wordpress.android.ui.posts.social.PostSocialConnection
+
+class EditPostSettingsJetpackSocialContainerComposeView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null
+) : AbstractComposeView(context, attrs) {
+    private var data: EditPostSettingsJetpackSocialContainerComposeViewData? = null
+
+    @Composable
+    override fun Content() {
+        AppThemeEditor {
+            data?.let {
+                EditPostSettingsJetpackSocialContainerView(
+                    trainOfIconsModels = it.trainOfIconsModels,
+                    postSocialConnectionList = it.postSocialConnectionList,
+                    shareMessage = it.shareMessage,
+                    remainingSharesMessage = it.remainingSharesMessage,
+                    subscribeButtonLabel = it.subscribeButtonLabel,
+                    onSubscribeClick = it.onSubscribeClick,
+                    connectProfilesButtonLabel = it.connectProfilesButtonLabel,
+                    onConnectProfilesCLick = it.onConnectProfilesCLick,
+                )
+            }
+        }
+    }
+
+    fun update(data: EditPostSettingsJetpackSocialContainerComposeViewData) {
+        this.data = data
+        disposeComposition()
```

Comment:
💡 Suggestion: instead of doing this imperatively by disposing the composition I think it would be better to store the `data` in a `MutableState` so `update()` can update that state and that would already cause a recomposition. [Here's an example](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/ui/compose/views/TrainOfAvatarsView.kt) of a Compose View Wrapper doing that. Does that make sense?

Reply:
Good idea 👍  if it's OK for you, I'll handle this in my next PR that implements the VM changes and starts using this View.

Reply:
Sounds good!

Reply:
[Done](https://github.com/wordpress-mobile/WordPress-Android/pull/18781/files#diff-3c86b940c933439b5e96091155d817bd64b0a658d5d8e477920ad3d66813d0cbR19)